packages/blade/src/components/Settings/Settings.stories.tsx

```diff
@@ -0,0 +1,1127 @@
+                Back
+              </BladeLink>
+            </ReactRouterDomLink>
+          </Box>
+        </Box>
+        <Box display="flex" justifyContent="flex-start" width="100%">
+          <Heading size="large" weight="semibold" color="surface.text.gray.normal">
+            User Profile
+          </Heading>
+        </Box>
+
+        <Box
+          width="100%"
+          marginTop={{
+            xs: 'spacing.7',
+            l: 'spacing.8',
+          }}
+        >
+          {/* Profile Details Card */}
+          <SubSectionCard
+            title="Profile details"
+            subtitle="These are your personal, user-level details"
+          >
+            <Box display="flex" flexDirection="column" gap="spacing.5">
+              {/* Name field */}
+              <Box
+                display="grid"
+                gridTemplateColumns={isMobile ? '150px 1fr' : '250px 1fr'}
+                alignItems="center"
+              >
+                <Text size="medium" weight="medium" color="surface.text.gray.muted">
+                  Name
+                </Text>
+                <Text size="medium" weight="semibold" color="surface.text.gray.subtle">
+                  Blade Team
+                </Text>
+              </Box>
+
+              {/* User role field */}
+              <Box
+                display="grid"
+                gridTemplateColumns={isMobile ? '150px 1fr' : '250px 1fr'}
+                alignItems="center"
+              >
+                <Text size="medium" weight="medium" color="surface.text.gray.muted">
+                  User role
+                </Text>
+                <Text size="medium" weight="semibold" color="surface.text.gray.subtle">
+                  Owner
+                </Text>
+              </Box>
+
+              {/* Email ID field */}
+              <Box
+                display="grid"
+                gridTemplateColumns={isMobile ? '150px 1fr' : '250px 1fr'}
+                alignItems="center"
+              >
+                <Text size="medium" weight="medium" color="surface.text.gray.muted">
+                  Email ID
+                </Text>
+                <Box display="flex" flexDirection="row" gap="spacing.2" alignItems="center">
+                  <Text size="medium" weight="semibold" color="surface.text.gray.subtle">
+                    owner@gmail.com
+                  </Text>
+                  <IconButton
+                    icon={EditIcon}
+                    size="medium"
+                    accessibilityLabel="Edit"
+                    // eslint-disable-next-line @typescript-eslint/no-empty-function
+                    onClick={() => {}}
+                  />
+                </Box>
+              </Box>
+
+              {/* Login password field */}
+              <Box
+                display="grid"
+                gridTemplateColumns={isMobile ? '150px 1fr' : '250px 1fr'}
+                alignItems="center"
+              >
```

Comment:
Let's try to refactor out components into reusable sections and give them meaningful names, if we keep using Boxes in pattern examples it will: 

- Get harder for AI to reason about the code and wont be able to structure properly
- And increases the tokens

`<Box display="grid" gridTemplateColumns={isMobile ? '150px 1fr' : '250px 1fr'}/>` this layout is being using many times, let's factor out into proper layout components.